src/components/layout/SiteLayout.tsx

```diff
@@ -50,10 +50,13 @@ const NavBar = () => {
         <NavLink href={"/2021/sponsors"} color={colors[3]}>
           Sponsors
         </NavLink>
-        <NavLink href={"/2021/awards"} color={colors[4]}>
+        <NavLink href={"/2021/sponsorlogistics"} color={colors[4]}>
```

Comment:
@immortalbard Should the sponsor logistics page be linked from the nav bar? I would think we could just send the link to this page directly to the sponsors rather than having be on the nav.

Reply:
Can it appear on the nav bar only when you're on the sponsor logistics page?

Reply:
That's honestly a little bit annoying to implement. Can we just skip it?

Reply:
Hi all - the link doesn't need to show up on the nav bar. Can you all add a hyper link on the "sponsors" tab? Have a text that reads "If you're a sponsor, please click here to find all logistics information on exhibiting and accessing your sponsor benefits."

Also - btw - I don't check github as often as I check slack. If you need feedback from me (sponsorship chair), please slack me directly.

Reply:
@travigd , can that text "If you're a sponsor, please click here to find all logistics information on exhibiting at the expo and accessing your benefits <weblink>" be added to the sponsor search page?

Reply:
@travigd  @techniic0l0r  Checking in - is this page considered done? can we have it hosted on the sponsors page?